### DISCUSSION TITLE
What is markdown?

### DISCUSSION THREAD

**Discussion by DanielKlenda-tech** [skills/.github#453] [Communicate using Markdown]
What is Markdown?

**Comment by aflacake** [comment]
Markdown is a [lightweight syntax](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for communicating on GitHub. You can format text to add a heading, lists, bold, italics, tables, and many other stylings.

Let's do this course:
[![start-course](https://user-images.githubusercontent.com/1221423/235727646-4a590299-ffe5-480d-8cd5-8194ea184546.svg)](https://github.com/new?template_owner=skills&template_name=communicate-using-markdown&owner=%40me&name=skills-communicate-using-markdown&description=My+clone+repository&visibility=public)
